kernel/sched/core.c

```diff
@@ -4644,6 +4644,21 @@ static __always_inline struct rq *
 context_switch(struct rq *rq, struct task_struct *prev,
 	       struct task_struct *next, struct rq_flags *rf)
 {
+#ifdef CONFIG_SYMBIOTE
+	unsigned int is_thread_sym_elevated;
+	unsigned int gs_low, gs_high;
+	unsigned long long kernel_gs;
+
+	is_thread_sym_elevated = next->symbiote_elevated;
+
+    if (is_thread_sym_elevated) {
```

Comment:
Add a comment in here to describe what you're trying to do exactly. Explain if there are any remaining unknowns or assumptions you're making